requirements-conda.txt

```diff
@@ -0,0 +1,109 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
+attrs
+backcall
+bleach
+certifi
+chardet
+click
+configobj
+cycler
+decorator
+defusedxml
+dill
+docutils
+h5py
+idna
+intel-openmp
+ipykernel
+ipython
+ipython_genutils
+ipywidgets
+jedi
+jinja2
+jsonpickle
+jsonschema
+jupyter
+jupyter_client
+jupyter_console
+jupyter_core
+kiwisolver
+libgpuarray
+libiconv
+libpng
+libsodium
+libstdcxx-ng
+libtiff
+libuuid
+libxcb
+libxml2
+lz4-c
+mako
+markupsafe
+matplotlib
+mistune
+mkl
+mkl-service
+more-itertools
+nbconvert
+nbformat
+ncurses
+notebook
+numpy
+openblas
+openssl
+packaging
+pandas
+pandoc
+pandocfilters
+pango
+parso
+patsy
+pcre
+pexpect
+pickleshare
+pip
+pixman
+prometheus_client
+prompt_toolkit
+pthread-stubs
+ptyprocess
+pygments
+pygpu
+pymc3
+pyparsing
+pyqt
+pyrsistent
+python
```

Comment:
This will (presumably) install python 3.7
Are you sure this won't result in any issues, given that the Theano-docs state that they support "Python == 2.7* or ( >= 3.4 and < 3.6 )"?
- http://deeplearning.net/software/theano/install_ubuntu.html

Reply:
The code compiled and ran fine up to the non-imported `gradient_descent` function. :) :raised_hands:

Reply:
Great :). Regarding Theano on Python 3.7: I haven't had a problem yet on my machine, and Theano is still maintained by the PyMC3 devs, so major issues should still be fixed. But I imagine incompabilities not impacting PyMC3 may eventually crop up. Because some other packages now only support >= 3.6, I've found that staying on 3.5 is for me more trouble than it's worth, but YMMV.

*Sinn* itself though should remain Python 3.5 compatible, and if it isn't that's a bug.